app-system/sources.list

```diff
@@ -3,7 +3,7 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 
 # A collection of fully FLOSS app definitions and FLOSS apps for Citadel.
-https://github.com/runcitadel/apps v3-beta
+https://github.com/runcitadel/apps v4-dev
```

Comment:
Make sure to change this back when merging

Reply:
Done